addons/packages/pinniped/0.12.1/bundle/examples/mc-idp-type-none.yaml

```diff
@@ -0,0 +1,26 @@
+#@data/values
```

Comment:
I included the examples..... tempted to suggest we should even have a little bash script to loop some ytt commands and validate expected outputs.